librz/core/cmd_analysis.c

```diff
@@ -6618,27 +6582,6 @@ static void _analysis_calls(RzCore *core, ut64 addr, ut64 addr_end, bool printCo
 					rz_io_read_at(core->io, op.jump, buf, 4);
 					isValidCall = memcmp(buf, "\x00\x00\x00\x00", 4);
 				}
-				if (isValidCall) {
-#if JAYRO_03
-					if (!analysis_is_bad_call(core, from, to, addr, buf, bufi)) {
-						fcn = rz_analysis_get_fcn_in(core->analysis, op.jump, RZ_ANALYSIS_FCN_TYPE_ROOT);
-						if (!fcn) {
-							rz_core_analysis_fcn(core, op.jump, addr, RZ_ANALYSIS_REF_TYPE_CALL, depth);
-						}
-					}
-#else
-					if (printCommands) {
-						rz_cons_printf("ax 0x%08" PFMT64x " 0x%08" PFMT64x "\n", op.jump, addr);
-						rz_cons_printf("af @ 0x%08" PFMT64x "\n", op.jump);
```

Comment:
you could substitute the "af" line with the `rz_core_analysis_function_add(core, NULL, op.jump, false)` while you are on it.